README.md

```diff
@@ -1,187 +1,159 @@
-```gradle
-repositories {
-    jcenter()
-    ...
-    maven { 
-        url 'https://jitpack.io' 
-    }
-}
-```
-
-Or, if you use Maven:
-```maven
-<repositories>
-    <repository>
-        <id>jitpack.io</id>
-        <url>https://jitpack.io</url>
-    </repository>
-</repositories>
-```
-
-For versions of Skript after dev37 you might need to add the paper-api repository to prevent build issues.
-
-```gradle
-maven { 
-    url 'https://repo.destroystokyo.com/repository/maven-public/' 
-}
-```
-
-Or, if you use Maven:
-```maven
-<repository>
-    <id>destroystokyo-repo</id>
-    <url>https://repo.destroystokyo.com/content/repositories/snapshots/</url>
-</repository>
-```
-
-Then you will also need to add Skript as a dependency.
-```gradle
-dependencies {
-    implementation 'com.github.SkriptLang:Skript:[versionTag]'
-}
-```
-
-An example of the version tag would be ```dev37c```.
-
-> Note: If Gradle isn't able to resolve Skript's dependencies, just [disable the resolution of transitive dependencies](https://docs.gradle.org/current/userguide/managing_transitive_dependencies.html#sub:disabling_resolution_transitive_dependencies) for Skript in your project.
-
-Or, if you use Maven:
-```
-<dependency>
-    <groupId>com.github.SkriptLang</groupId>
-    <artifactId>Skript</artifactId>
-    <version>[versionTag]</version>
-</dependency>
-```
-
-> Note that these repositories are provided as-is, for now. I cannot currently spend time to add nice, but not mandatory, features like Javadoc.
-
-## Relevant Links
-* [SkUnity Forums](https://forums.skunity.com/)
-* [Original Skript at BukkitDev](https://dev.bukkit.org/bukkit-plugins/skript/) (inactive)
-* [Addon Releases @SkUnity](https://forums.skunity.com/forums/addon-releases/)
-* [Skript Chat Discord Invite](https://discord.gg/0lx4QhQvwelCZbEX)
-* [Skript Hub](https://skripthub.net/)
-
-Note that these resources are not maintained by me. If you notice something wrong with them, do not contact me.
-
-## Developers
-Current team behind Skript:
-
-* [bensku](https://github.com/bensku) (Skript maintainer/developer)
-* [Snow-Pyon](https://github.com/Snow-Pyon) (Skript developer)
-* [Pikachu920](https://github.com/Pikachu920) (Skript developer)
-* [Nicofisi](https://github.com/Nicofisi) (Skript developer)
-* [TheBentoBox](https://github.com/TheBentoBox) (issue tracker manager)
-
-Also, of course, we should thank [Njol](https://github.com/Njol) for creating
-Skript and [Mirreski](https://github.com/Mirreski) for maintaining it for a
-long time.
-
-In addition to that, Skript has received a lot of pull requests over time.
-You can find all contributors [here](https://github.com/bensku/Skript/graphs/contributors).
-
-All code is owned by it's writer, licensed for others under GPLv3 (see LICENSE)
-unless otherwise specified.
+# Skript [![Build Status](https://travis-ci.org/SkriptLang/Skript.svg?branch=master)](https://travis-ci.org/SkriptLang/Skript)
```

Comment:
Please avoid changing README for now. It is not really related to templates.

Reply:
![image](https://user-images.githubusercontent.com/19474570/47934293-01df7200-dee8-11e8-865b-ea8b8a402904.png)
🤷‍♀️